CONTRIBUTING.md

```diff
@@ -0,0 +1,57 @@
+Want to contribute? Great! First, read this page (including the small print at
```

Comment:
This shows up when filing issues, too. Please make the top section about how to file good issues.

* Make sure to include the URL
  * Use the link on the page, if possible, the URL will be included
* ...make sure you have a descriptive title
* ... etc etc